modules/ossm-configuring-istio-ha-autoscaling.adoc

```diff
@@ -0,0 +1,72 @@
+// This procedure is used in the following assembly:
+// * service-mesh-docs-main/install/ossm-installing-service-mesh.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="ossm-configuring-istio-ha-autoscaling_{context}"]
+= Configuring Istio HA by using autoscaling 
+
+Configure the {istio} control plane in High Availability (HA) mode to prevent a single point of failure, and ensure continuous mesh operation even if one of the `istiod` pods fails. Autoscaling defines the minimum and maximum number of {istio} control plane pods that can operate. {ocp-product-title} uses these values to scale the number of control planes in operation based on resource utilization, such as CPU or memory, to efficiently respond to the varying number of workloads and overall traffic patterns within the mesh.
+
+.Prerequisites
+
+* You are logged in to the {ocp-product-title} web console as a user with the `cluster-admin` role.
+
+* You installed the {SMProductName} Operator.
+
+* You deployed the {istio} resource.
+
+.Procedure
+
+. In the {ocp-product-title} web console, click *Installed Operators*.
+
+. Click {SMProductName} 3 Operator.
+
+. Click *Istio*.
+
+. Click the name of the {istio} installation. For example, `default`.
+
+. Click *YAML*.
+
+. Modify the {istio} custom resource.
++
+.Example configuration
+[source,yaml,subs="attributes,verbatim"]
+----
+apiVersion: sailoperator.io/v1
+kind: Istio
+metadata:
+  name: default
+spec:
+  namespace: istio-system
+  values:
+    pilot:
+      autoscaleMin: 2 # <1>
+      autoscaleMax: 5 # <2>
+      cpu:
+        targetAverageUtilization: 80 # <3>
+      memory:
+        targetAverageUtilization: 80 # <4>
+----
+<1> Defines the minimum number of {istio} control plane replicas that always run. 
+<2> Defines the maximum number of {istio} control plane replicas, allowing for scaling based on load. To support HA, there must be at least two replicas.
+<3> Defines the target CPU utilization for autoscaling to 80%. If the average CPU usage exceeds this threshold, the Horizontal Pod Autoscaler (HPA) automatically increases the number of replicas.
+<4> Defines the target memory utilization for autoscaling to 80%. If the average memory usage exceeds this threshold, the HPA automatically increases the number of replicas.
+
+.Verification
+
+. Verify the status of the {istio} control pods by running the following command:
```

Comment:
```suggestion
* Verify the status of the {istio} control pods by running the following command:
```
Since this is a single step procedure, I think it is okay to not make it a numbered-list. Here's the [guideline](https://redhat-documentation.github.io/supplementary-style-guide/#single-step-procedures).